src/test/java/com/brianmcmichael/sagu/AppPropertiesTest.java

```diff
@@ -27,11 +27,14 @@
 
     private String propertiesResourcePath;
     private String resourceDir;
-
+	private static final boolean IS_WINDOWS = System.getProperty( "os.name" ).contains( "indow" );
```

Comment:
Could you please replace tabs with 4 spaces to match the rest of the code?

Reply:
ok